### ISSUE TITLE
Rename AboutPage to more generic title?

### ISSUE THREAD

**Issue by heymonkeyriot** [wagtail/bakerydemo#78]
I asked this question on PR #76

> One thought I had with this, especially if we remove the location relationship as per #54, is whether we should describe this as an "about page". On most of the Wagtail sites I've worked on we have some sort of standard page that have no special attributes but can be used to create these sort of pages (be they about pages, privacy policies etc.) Would we be better off changing the name of this model to something like StandardPage?

**Comment by shacker** [comment]
+1 I think that's a good idea. We don't want to imply that devs have to create a new model for every variant of standard-issue pages. I'll take this one tonight.

**Comment by heymonkeyriot** [comment]
Awesome!

**Comment by shacker** [comment]
Soon realized that we already have `base.BasePageFieldsMixin` which defines introduction, image, and body as `blocks.BaseStreamBlock`, which left me wondering whether we really need `StandardPage`, or whether we're better off sticking with custom models inheriting from `base.BasePageFieldsMixin`. Two different techniques for abstraction  - which is better? Mixins have better composability, but `StandardPage` is a common pattern. 

Thoughts?

**Comment by shacker** [comment]
(actually I already moved `body` into BasePageFieldsMixin` - you guys won't see that yet.

**Comment by shacker** [comment]
This turned interesting.

I made an executive decision and decided to create a StandardPage, definining body, image and introduction. All good.

Then I went to remove the `BasePageFieldsMixin` and make all the app.models that use it subclass StandardPage instead, since those were redundant.  Ran into a big migration problem. `makemigrations` wants me to define a default value for `standardpage_ptr`.  I can't make it null or optional since it's a subclass, not a FK relation. I can't specify 1 since I can't guarantee that exists. I can't use `StandardPage.objects.first()` since the migration making tool doesn't recognize the import. It seems the only option is to wipe the db and start over but don't want to do that. Stuck.

Now feeling it would be better to keep `BasePageFieldsMixin` and ditch StandarPage. Though I could wipe the db if anyone feels strongly about it one way or another.

Good object lesson in why mixins are more flexible than StandardPage-style approach,
